### ISSUE TITLE
Support for Goodwe SBP Retrofit

### ISSUE THREAD

**Issue by corty8** [ai-republic/bms-to-inverter#83]
Hi there

Is it possible to add the Goodwe SBP (GW5000S-BP AC Coupled Retrofit Charger?

Thanks

**Comment by ai-republic** [comment]
Sure, do you have the specification? I see it supports CAN and RS485. Better is CAN.

**Comment by corty8** [comment]
does this help
[GoodweCAN_BUS_Protocol.pdf](https://github.com/user-attachments/files/17878071/GoodweCAN_BUS_Protocol.pdf)

**Comment by ai-republic** [comment]
Yes, this is actually the same as the Pylontec CAN protocol. Could you please test it with the _PYLON_CAN_ BMS binding?

**Comment by corty8** [comment]
ok great, I am waiting on a couple of components to be delivered and I will test it and let you know

**Comment by ai-republic** [comment]
@corty8 did you get a chance to test it?

**Comment by corty8** [comment]
> @corty8 did you get a chance to test it?

no, still haven't been able to
The wifi module on the inverter got damaged when it was being moved and I have not been able to source the correct one
have ordered 2 now and they have sent the wrong one on both occasions
